packages/manager/modules/account-migration/src/constants.js

```diff
@@ -0,0 +1,58 @@
+export const FAQ_LINK = {
```

Comment:
```suggestion
const FI_FAQ_LINK = 'https://www.ovh-hosting.fi/ukk/';
export const FAQ_LINK = {
CZ: FI_FAQ_LINK,
DE: FI_FAQ_LINK
```
avoid string duplication

Reply:
The links for the other subsidiaries are not required now. They can be added when required. Hence the redundant links have been removed